src/sentry/charts/types.py

```diff
@@ -0,0 +1,11 @@
+from enum import Enum
+
+
+class ChartType(Enum):
+    """
+    This enum defines the chart styles we can render.
+
+    This directly maps to the chartcuterie configuration in the frontend code, see
+    app/chartcuterieConfig.tsx."""
```

Comment:
Just checking there's a comment in the FE file that says to keep this file in sync